README.md

```
@@ -93,11 +94,12 @@ There is a helpful script to setup a new SDK
 ```shell
 make setup-new-sdk
 ```
-
-1. It will ask you for a an SDK ID, use something like: go, js, dotnet, java, etc...
-2. It will ask you for a [valid generator](https://github.com/OpenAPITools/openapi-generator/blob/master/docs/generators.md)
-3. Then in will initialize all the files, you will need to add the configuration specific to that sdk
-4. Now you can run `make build-client-${SDK_ID}` to generate the SDK
+1. It will ask you for:
+    * an SDK ID: (e.g. `go`, `js`, `dotnet`, `java`) - this will become the name of the SDK.
+    * OpenAPI Generator name for the `-g` flag (e.g., `ruby`, `go`, `python-legacy`) - this must be a [valid generator](https://github.com/OpenAPITools/openapi-generator/blob/master/docs/generators.md)
+    * Generator Template Path - Directory in OpenAPI Generator repo (usually same as generator, but can differ - e.g., `ruby-client` for `ruby`) - the template must exist [here](https://github.com/OpenAPITools/openapi-generator/tree/master/modules/openapi-generator/src/main/resources)
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)